src/jit/morph.cpp

```diff
@@ -13653,6 +13653,7 @@ GenTree* Compiler::fgMorphSmpOpOptional(GenTreeOp* tree)
                         /* The target is used as well as being defined */
                         if (op1->OperIsLocal())
                         {
+                            op1->gtFlags &= GTF_VAR_USEDEF;
```

Comment:
This will clear all flags on op1 leaving aside GTF_VAR_USEDEF.  From the description of the fix, the intent is to clear this flag.  Shouldn't this be

op1->gtFlags &= ~GTF_VAR_USEDEF;